casper/src/main/scala/coop/rchain/casper/engine/LastFinalizedStateTupleSpaceRequester.scala

```diff
@@ -138,22 +142,40 @@ object LastFinalizedStateTupleSpaceRequester {
         // Import chunk to RSpace
         _ <- Stream
               .eval {
-                // Get importer
-                val importer = RSpaceStateManager[F].importer
+                // Transform values from ByteString to ByteVector
+                val historyItemsWithByteVector = historyItems.map {
+                  case (k, v) => (k, ByteVector(v.toByteArray))
+                }
+                val dataItemsWithByteVector = dataItems.map {
+                  case (k, v) => (k, ByteVector(v.toByteArray))
+                }
+
                 for {
+                  // Validate received tuple space items
+                  _ <- Lib.time("VALIDATE STATE CHUNK")(
```

Comment:
Just curious that is there a plan to remove this one?

Reply:
Yes, but before we have easy diagnostics setup, I'll transform this to Log output.

Reply:
I transformed this to [Stopwatch](https://github.com/rchain/rchain/blob/52aa8bc5/shared/src/main/scala/coop/rchain/shared/Stopwatch.scala) which can measure duration of some block of code.